linera-execution/src/execution_state_actor.rs

```diff
@@ -97,6 +105,59 @@ where
                 callback.respond(balance);
             }
 
+            OwnerBalance { owner, callback } => {
+                let balance = *self.system.balances.get_mut_or_default(&owner).await?;
+                callback.respond(balance);
+            }
+
+            Transfer {
+                source,
+                destination,
+                amount,
+                callback,
+            } => {
+                let operation_context =
+                    operation_context.expect("Operation Context shouldn't be null here");
+
+                self.system
```

Comment:
Let's pass the authentication from the calling application inside the request and then create a fake `OperationContext` for now.